### PR TITLE
Replaced python server with node server

### LINKED ISSUE #10: Convert python server to node.js express app
I started this project with a Python server so I could run NuPIC on the server. Since then, I have moved all that to another server, so this project can be converted to 100% JS.

This task is to remove `server.py` and replace with `index.js` that does exactly the same thing.

### PR DESCRIPTION
**Pull request by rikkertkoppes**
This closes #10 

Some remarks:

The express-handlebars module uses a different file structure than currently used, see https://github.com/ericf/express-handlebars#basic-usage

I added a bit of configuration to keep the current structure intact, but it might be worth changing.

### CODE REVIEW

**Comment by rhyolight** [comment]
@rikkertkoppes Thanks for this. I will try it out today.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+var GIF_PATH = "static/data/gifdata/";
+
+//use handlebars render engine using the existing locations
+app.engine('.html', exphbs({
+    extname: '.html',
+    defaultLayout: 'layout',
+    layoutsDir: "tmpl/"
+}));
+app.set('view engine', '.html');
+app.set('views', 'html/')
```
Comment (rhyolight) [inline index.js]: semicolon

**Review by rhyolight — CHANGES_REQUESTED** [review]
Nicely done! I found one missing semicolon. ;)

**Comment by rikkertkoppes** [comment]
gone

**Review by rhyolight — APPROVED** [review]

**Comment by rhyolight** [comment]
Works great 👍

**Pull request merged**